ziti/enroll/enroll.go

```diff
@@ -299,11 +299,27 @@ func enrollUpdb(username, password string, token *ziti.EnrollmentClaims, caPool
 		_, _ = body.Set(username, "username")
 	}
 
-	resp, err := client.Post(token.EnrolmentUrl(), "application/json", bytes.NewBuffer(body.EncodeJSON()))
+	enrollmentUrls := token.EnrolmentUrls()
+
+	var resp *http.Response
+	var err error
+	for _, enrollmentUrl := range enrollmentUrls {
+		resp, err = client.Post(enrollmentUrl, "application/json", bytes.NewBuffer(body.EncodeJSON()))
+
```

Comment:
Do you want to break after the first non-erroring response?

Reply:
Why yes it should.